README.md

```diff
@@ -6,12 +6,17 @@ There are two supported ways to install the module, either via [Composer](https:
 
 ### Installation Using Composer
 
-To install the AddresssFinder module for Magento 1, simply run:
+To install the AddresssFinder module for Magento 1, run:
 
 ```
 composer require addressfinder/module-magento1
+composer config extra.magento-root-dir "MAGENTO INSTALLATION DIRECTORY"
+composer require magento-hackathon/magento-composer-installer:*
+composer run-script post-install-cmd -vvv -- --redeploy
```

Comment:
@bencorlett here is the magento-hackathon/magento-composer-installer stuff

Reply:
Missed this!

This doesn’t seem necessary as it happens. We should be listing the composer installer as one of our dependencies as opposed to the project that requires it. 

I’ll come up with a PR with a proposed solution shortly :)